suite-native/module-trading/src/selectors/exchangeSelectors.ts

```diff
@@ -98,3 +130,86 @@ export const selectGroupedExchangeQuotes = createMemoizedSelector(
         return groups;
     },
 );
+
+export const selectExchangeAccountsWithTokensSectionList = createExchangeMemoizedSelector(
+    [
+        selectVisibleDeviceAccounts,
+        selectTokenDefinitions,
+        selectCurrentFiatRates,
+        selectLocalCurrency,
+        state => state,
+    ],
+    (accounts: Account[], tokenDefinitions, fiatRates, localCurrency, state) =>
+        accounts
+            .map<SectionListData<MyAsset, Account>[number]>((account: Account) => {
+                const networkTokenDefinitions = getSimpleCoinDefinitionsByNetwork(
+                    tokenDefinitions,
+                    account.symbol,
+                );
+
+                const knownTokens = filterKnownTokens(
+                    networkTokenDefinitions,
+                    account.symbol,
+                    account.tokens ?? [],
+                );
+
+                const tokensWithBalance = knownTokens.filter(
+                    token => parseFloat(token?.balance ?? '0') > 0,
+                );
+
+                const tokens: MyAsset[] = tokensWithBalance.map(token => {
+                    const fiatRateKey = getFiatRateKey(
+                        account.symbol,
+                        localCurrency,
+                        token.contract as TokenAddress,
+                    );
+                    const rate = fiatRates?.[fiatRateKey]?.rate;
+                    const fiatBalance =
+                        rate && token.balance ? toFiatCurrency(token.balance, rate) : null;
+
+                    const tokenSymbol = selectAccountTokenSymbol(
+                        state as unknown as TokensRootState,
+                        account.key,
+                        token.contract as TokenAddress,
+                    );
+
+                    return {
+                        symbol: account.symbol,
+                        name: token.name ?? '',
```

Comment:
nit: just for sure there is something
```suggestion
                        name: token.name ?? token.symbol ?? '',
```

Reply:
Fixed in 1adce137ce